CASSANDRA.md

```diff
@@ -307,9 +300,34 @@ object Phantom extends App {
 }
 ```
 
-Phantom allows the user certain level of composability, but it gets a bit verbose due to the nature of the DSL. 
+Phantom allows the user certain level of composability, but it gets a bit verbose due to the nature of the DSL.
+However, phantom does not encourage this inherent duplication of concern because it doesn't add any value
+to an application, and instead generally prefers to duplicate from the start.
+
+On a character count, this is likely to be the shortest approach of all frameworks.
```

Comment:
@fwbrasil I don't disagree, I just don't understand the advocated advantage of composability here.